src/sentry/static/sentry/app/views/settings/project/projectProcessingIssues.tsx

```diff
@@ -199,50 +211,55 @@ class ProjectProcessingIssues extends React.Component {
     });
   };
 
-  renderDebugTable = () => {
-    let body;
+  renderDebugTable() {
+    let body: React.ReactNode;
     if (this.state.loading) {
       body = this.renderLoading();
     } else if (this.state.error) {
       body = <LoadingError onRetry={this.fetchData} />;
     } else if (
-      this.state.processingIssues.hasIssues ||
-      this.state.processingIssues.resolveableIssues ||
-      this.state.processingIssues.issuesProcessing
+      this.state.processingIssues &&
+      (this.state.processingIssues.hasIssues ||
+        this.state.processingIssues.resolveableIssues ||
+        this.state.processingIssues.issuesProcessing)
     ) {
       body = this.renderResults();
     } else {
       body = this.renderEmpty();
     }
 
     return body;
-  };
+  }
+
+  renderLoading() {
+    return (
+      <div className="box">
+        <LoadingIndicator />
+      </div>
+    );
+  }
+
+  renderEmpty() {
+    return (
+      <Panel>
+        <EmptyStateWarning>
```

Comment:
Shouldn't we use `PanelBody` here as well?

Reply:
I didn't want to alter the page contents much, just trying to get the typescript changes in.